### PR TITLE
Adds Head of Service, multiple job tweaks + updated gitignore

### PR DESCRIPTION
**Pull request by Jolly-66**
## About The Pull Request
Adds Head of Service, also tweaks all service jobs to state that they respond TO the HoSv (Head of Service).
HoSv responds to the HoP
Quartermaster was made into a head a head and responds directly to the HoP. Cargonia responds to the QM now.
They also should now have bridge access too.

## Why It's Good For The Game
https://cdn.discordapp.com/attachments/802230528785776651/805825600458981456/unknown.png

## Changelog
:cl:
add: Added Head of Service
tweak: Minor service and cargo department tweaks. QM is now a head and with access.
/:cl:

### CODE REVIEW

**Comment by Jolly-66** [comment]
git ignore should now just block data now for this branch

i'll still push the data folder to the repo every so often in the event i lose my local data

**Comment by Jolly-66** [comment]
i should also mention this doesn't neuter the hop controlling servier and cargo
with the chain of command chart, hes more of a helicopter parent over the departments to make sure they're not on fire (or armed with guns)
he can still approve and deny shit for them, which could be a thorn in the ass to his two heads

File: code/modules/jobs/job_types/bartender.dm [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /datum/job/bartender
 	title = "Bartender"
-	department_head = list("Head of Personnel")
+	department_head = list("Head of Service")
```
Comment (MrMelbert) [inline code/modules/jobs/job_types/bartender.dm:3]:
```suggestion
	department_head = list("Head of Service") // NON-MODULE CHANGE
```
I'm not gonna go down and do it to all of these but these should all be commented

File: code/modules/jobs/job_types/bartender.dm [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 	faction = "Station"
 	total_positions = 1
 	spawn_positions = 1
-	supervisors = "the head of personnel"
+	supervisors = "the head of service"
```
Comment (MrMelbert) [inline code/modules/jobs/job_types/bartender.dm:7]:
```suggestion
	supervisors = "the head of service" // NON-MODULE CHANGE
```
same as above

File: jollystation_modules/code/game/objects/items/devices/radio/encryptionkey.dm [hunk 3]
```diff
@@ -0,0 +1,4 @@
+/obj/item/encryptionkey/heads/hosv
+	name = "\proper the head of service's encryption key"
+	icon_state = "hop_cypherkey"
+	channels = list(RADIO_CHANNEL_SERVICE = 1, RADIO_CHANNEL_COMMAND = 1)
```
Comment (MrMelbert) [inline jollystation_modules/code/game/objects/items/devices/radio/encryptionkey.dm:4]:
```suggestion
	channels = list(RADIO_CHANNEL_SERVICE = 1, RADIO_CHANNEL_COMMAND = 1)
	
```

File: jollystation_modules/code/game/objects/items/devices/radio/headset.dm [hunk 4]
```diff
@@ -0,0 +1,5 @@
+/obj/item/radio/headset/heads/hosv
+	name = "\proper the head of service's headset"
+	desc = "The headset of the ex janitor-lawyer-clown-botanist-mime-chef."
+	icon_state = "com_headset"
+	keyslot = new /obj/item/encryptionkey/heads/hosv
```
Comment (MrMelbert) [inline jollystation_modules/code/game/objects/items/devices/radio/headset.dm:5]:
```suggestion
	keyslot = new /obj/item/encryptionkey/heads/hosv
	
```

File: jollystation_modules/code/modules/jobs/job_types/head_of_service.dm [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+						ACCESS_RC_ANNOUNCE, ACCESS_KEYCARD_AUTH, ACCESS_GATEWAY, ACCESS_MINERAL_STOREROOM, ACCESS_AUX_BASE, ACCESS_TELEPORTER)
+	minimal_access = list(ACCESS_COURT,
+						ACCESS_MEDICAL, ACCESS_PSYCHOLOGY, ACCESS_ENGINE, ACCESS_CHANGE_IDS, ACCESS_AI_UPLOAD, ACCESS_EVA, ACCESS_HEADS,
+						ACCESS_ALL_PERSONAL_LOCKERS, ACCESS_MAINT_TUNNELS, ACCESS_BAR, ACCESS_JANITOR, ACCESS_CONSTRUCTION, ACCESS_MORGUE,
+						ACCESS_CREMATORIUM, ACCESS_KITCHEN, ACCESS_CARGO, ACCESS_MAILSORTING, ACCESS_QM, ACCESS_HYDROPONICS, ACCESS_LAWYER,
+						ACCESS_MECH_MINING, ACCESS_MECH_ENGINE, ACCESS_MECH_SCIENCE, ACCESS_MECH_SECURITY, ACCESS_MECH_MEDICAL,
+						ACCESS_THEATRE, ACCESS_CHAPEL_OFFICE, ACCESS_LIBRARY, ACCESS_RESEARCH, ACCESS_MINING, ACCESS_VAULT, ACCESS_MINING_STATION,
+						ACCESS_HOP, ACCESS_RC_ANNOUNCE, ACCESS_KEYCARD_AUTH, ACCESS_GATEWAY, ACCESS_MINERAL_STOREROOM, ACCESS_AUX_BASE, ACCESS_TELEPORTER)
+	paycheck = PAYCHECK_COMMAND
+	paycheck_department = ACCOUNT_SRV
```
Comment (MrMelbert) [inline jollystation_modules/code/modules/jobs/job_types/head_of_service.dm:34]: Might want to move the Head of Personnel off of `ACCOUNT_SRV`, maybe onto `ACCOUNT_SEC` like the captain or `ACCOUNT_CAR` to make the HoP the station's budget manager

Comment (MrMelbert) [reply jollystation_modules/code/modules/jobs/job_types/head_of_service.dm:34]: For the record `paycheck_department` is what determines if you get discount of vendors
For heads of staff, it also allows you to use the department budget to buy things in cargo

**Review by MrMelbert — CHANGES_REQUESTED** [review]
Mostly just comments and newlines, otherwise good stuff

**Pull request merged**